diff --git a/node_modules/@expo/webpack-config/webpack/addons/withDevServer.js b/node_modules/@expo/webpack-config/webpack/addons/withDevServer.js
index 1234567..abcdefg 100644
--- a/node_modules/@expo/webpack-config/webpack/addons/withDevServer.js
+++ b/node_modules/@expo/webpack-config/webpack/addons/withDevServer.js
@@ -198,6 +198,9 @@ function createDevServer(env, { allowedHost, proxy } = {}) {
         // `proxy` is run between `before` and `after` `webpack-dev-server` hooks
         proxy,
         setupMiddlewares(middlewares, devServer) {
+            // Suppress deprecation warnings
+            if (process.noDeprecation === undefined) {
+                process.noDeprecation = true;
+            }
             (0, assert_1.default)(devServer, 'webpack-dev-server is not defined');
             middlewares.unshift(
             // Keep `evalSourceMapMiddleware`
